benchmarks/benchmarks/stats.py

```diff
@@ -95,6 +95,29 @@ def time_distribution(self, distribution, properties):
     time_distribution.version = "fb22ae5386501008d945783921fe44aef3f82c1dafc40cddfaccaeec38b792b0"
 
 
+class FrozenDistribution(Benchmark):
+    params = [
+            ['multivariate_normal'],
+            ['rvs'],
+            [1000]
+    ]
+    param_names = ['distribution', 'method', 'dimension']
+
+    def setup(self, distribution, method, dim):
+        np.random.seed(4321)
+        if distribution == 'multivariate_normal':
+            mean = np.random.rand(dim)
+            eigvals = np.random.rand(dim)
+            eigvals *= dim / np.sum(eigvals)
+            cov = stats.random_correlation.rvs(eigvals)
+            self.dist = stats.multivariate_normal(mean, cov)
+
+    def time_frozen_distribution(self, distribution, method, dim):
+        if distribution == 'multivariate_normal':
+            if method == 'rvs':
+                self.dist.rvs(100)
```

Comment:
Looks like this was written to invite other frozen distributions to join. It's more complicated than it needs to be for the single benchmark you're adding, but I suppose it makes sense if that was your intent.

Reply:
That was indeed my intention, given that a very similar fix can be made for the multivariate t distribution.